Resources/config/routing/redirect.xml

```diff
@@ -5,6 +5,6 @@
     xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
 
     <route id="hwi_oauth_service_redirect" path="/{service}">
-        <default key="_controller">HWI\Bundle\OAuthBundle\Controller\ConnectController::redirectToServiceAction</default>
+        <default key="_controller">HWI\Bundle\OAuthBundle\Controller\RedirectToServiceController::redirectToServiceAction</default>
```

Comment:
Those route changes must be note in `upgrade` file or something.

Reply:
mm the only thing I can see in the docs is that you have to import the `xml` files, so this shouldn't be a problem, I'm not sure where this should be mentioned